RATIONALE.md

```diff
@@ -65,4 +65,102 @@ KQueen supplies the backend API for provider-agnostic cluster management. It ena
 * **Update** - install newer version of Kubernetes
 * **Autoscale** - watch Kubernetes scheduler or pods and start new minions when all existing minions are fully utilized
 * **Manage addons** - enable or disable cluster addons
-
+* **Members managing** - Its possible to define own mail server or use default to provide user registering into Kqueen.
+Configure mail service in docker-compose.production.yml:
+```
+  mail:
+    image: modularitycontainers/postfix
+    volumes:
+      - /var/spool/postfix:/var/spool/postfix
+      - /var/mail:/var/spool/mail
+    environment:
+      MYHOSTNAME: 'mail'
+
+ ```
+Configure following variables in kqueen ui service:
+```
+  KQUEENUI_MAIL_SERVER: mail
+  KQUEENUI_MAIL_PORT: 10025
+```
+Pay attention that volume-mapping for mail containers is additional feature. It used for store mailing history and forward additional postfix mail configuration (it should configured properly on local machine),
+otherwise its possible to run mail server without volume-mapping. After all configuration steps, user should invite members using email notification. Member should receive mail with activation link to Kqueen service and possibility of password-setting.
```

Comment:
I suggest to move last to sentences to the top of this section, to clarify to the user why mail server is needed at all